server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -88,12 +97,21 @@ export class ScriptoriumLambda implements IPartitionLambda {
         Promise.all(allProcessed).then(
             () => {
                 this.current.clear();
+                if (this.telemetryEnabled) {
+                    metric.setProperty("processingComplete", true);
```

Comment:
Maybe here and below you can just add a `?` instead of having the `if` statement? Example:

`metric?.setProperty("processingComplete", true);`